docs: add cluster profile variables PEM-8647 (#8332)
spectrocloud/librarium: docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/define-profile-variables/create-cluster-profile-variables.md

    | IPv4 CIDR  | A CIDR block of IP addresses that follow the IPv4 standard.                                            |
    | IPv6       | Valid representation of an IPv6 address.                                                               |

> 9.  If you have selected the **Dropdown** input type in Step 7, use **+ Add dropdown option** to add dropdown options.
>     The **Add dropdown option** window appears. The following table describes the available configuration fields. You
>     must provide at least one dropdown option.

Reviewer: ```suggestion
9.  If you selected **Dropdown** for the **Input type** in step 7, select **Add dropdown option**.
    The following table describes the available configuration fields. You
    must provide at least one dropdown option.
```

    | **Field**       | **Description**                                                                                                |
    | --------------- | -------------------------------------------------------------------------------------------------------------- |

>     | **Value**       | The value of the dropdown option. This is a mandatory field.                                                   |

Reviewer: ```suggestion
    | **Value**       | Value of the dropdown option. This field is required.                                                   |
```

>     | **Label**       | Display name for the dropdown option. If no label is provided, then the **Value** is used as the display name. |

Reviewer: ```suggestion
    | **Label**       | Display name for the dropdown option. If no label is provided, the **Value** is used as the display name. |
```

    | **Parameter**               | **Description**                                                                                                                                                                                                                                 |
    | --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | **Custom input validation** | Validate the cluster profile variable definition against a regular expression based on the [regexp](https://pkg.go.dev/regexp) engine.                                                                                                          |
    | **Required**                | Require a value for the cluster profile variable.                                                                                                                                                                                               |

>     | **Default value**           | Set a default value for the cluster profile variable. If the input type is **Dropdown**, then a default value must be specified using one of the configured dropdown options.                                                                   |

Reviewer: ```suggestion
    | **Default value**           | Set a default value for the cluster profile variable. If the **Input type** is **Dropdown**, a default value must be specified using one of the configured dropdown options.                                                                   |
```
